tools/aa/commands/InitCommand.java

```diff
@@ -86,11 +97,42 @@ public boolean run(String[] args) {
       System.out.println(e.getMessage());
       System.out.println("Input flags:");
       Flags.printUsage();
+      revertChanges();
       return false;
     } catch (Exception e) {
+      revertChanges();
       e.printStackTrace();
     }
     return true;
   }
+
+  private void revertChanges() {
```

Comment:
I think in this case, init exits when `basePath.get()` contains something, so all we need to check is whether it existed.
If it did, leave it empty. If it didn't, delete it and all of its contents.

Reply:
Done.